src/features/cookies/components/CookiesSettings.native.test.tsx

```diff
@@ -19,9 +19,12 @@ describe('<CookiesSettings/>', () => {
     })
 
     essentialToggle && fireEvent.press(essentialToggle)
-    expect(essentialToggle?.props.accessibilityState).toEqual({
-      disabled: true,
-      checked: true,
+
+    await waitFor(() => {
+      expect(essentialToggle?.props.accessibilityState).toEqual({
+        disabled: true,
+        checked: true,
+      })
```

Comment:
le test peu échouer pour plusieurs raisons : 
* 2 `expect`s dans un meme test
* une condition dans le test

dans le futur, lors d'une modification, s'il y a une régression et qu'il échoue ça sera plus difficile de savoir pourquoi ça a échoué : il faudra creuser précisemment quelle est la raison parmi les 3 possibles pour laquelle il échoue

un code smell pour détecter les tests qui vérifient plusieurs choses d'un coup : est-ce que la règle du test contient un `and`

je recommande de découper ce test afin d'avoir un expect par test et pas de condition dans le test

Reply:
On a pas besoin de tester le clic comme on sait qu'il est disabled. Donc je peux supprimer le deuxième expect